app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -710,18 +735,6 @@ class PinPasswordActivityTest {
     return visibleActivityName == T::class.java.name
   }
 
-  private inline fun <reified T : Activity> waitUntilActivityVisible() {
```

Comment:
Looks like there are few unused methods which were previously used by this method. Need to remove.

Reply:
I found 2 such methods 'isVisible' and 'getCurrentActivity' and removed them which resulted in many unused imports so I removed those as well. I just had a confusion, two unused imports are also left
```
private const val TIMEOUT = 1000L
private const val CONDITION_CHECK_INTERVAL = 100L
```
Should I remove these two as well ?

Reply:
@anandwana001 I found 2 such methods 'isVisible' and 'getCurrentActivity' and removed them which resulted in many unused imports so I removed those as well. I just had a confusion, two unused imports are also left
```
private const val TIMEOUT = 1000L
private const val CONDITION_CHECK_INTERVAL = 100L
```
Should I remove these two as well ?

Reply:
yes, as these are of no use we can remove this.

Reply:
Okay

Reply:
Done